plugins/inputs/sqlserver/azuresqlqueries.go

```diff
@@ -1196,3 +1320,261 @@ SELECT
 	,DATABASEPROPERTYEX(DB_NAME(), 'Updateability') as replica_updateability
 FROM sys.dm_os_schedulers AS s
 `
+
+const sqlAzureMIPartQueryPeriod = `
```

Comment:
This is a sizable query, have we thought of limiting what we retrieve? Aka take for example MI with 20 databases, creating a temp table with everything in query store could be expensive? Should you get like the top 50 from each?